src/app/components/merge/MergedContactSummary.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Block, Icon } from 'react-components';
+
+import { formatAdr } from '../../helpers/property';
+import { getFirstValue, getAllValues } from '../../helpers/properties';
+
+const MergedContactSummary = ({ properties = [] }) => {
+    const name = getFirstValue(properties, 'fn');
+    const emails = getAllValues(properties, 'email');
+    const tels = getAllValues(properties, 'tel');
+    const adrs = getAllValues(properties, 'adr');
+    const orgs = getAllValues(properties, 'org');
+    const notes = getAllValues(properties, 'note');
+
+    const summary = [
+        [{ prop: 'Name', icon: 'contact', component: name }],
+        emails.length &&
```

Comment:
Are the checks for `${array}.length` necessary here? If they're not guaranteed to be an array I think it's better to use `Array.isArray(array)`